easybuild/tools/module_generator.py

```diff
@@ -226,6 +302,10 @@ class ModuleGeneratorTcl(ModuleGenerator):
     LOAD_REGEX = r"^\s*module\s+load\s+(\S+)"
     LOAD_TEMPLATE = "module load %(mod_name)s"
 
+    def __init__(self, *args, **kwargs):
+        """ModuleGeneratorTcl constructor."""
+        super(ModuleGeneratorTcl, self).__init__(*args, **kwargs)
```

Comment:
Well, at least now it is **consistently** useless. Before one of the classes had an useless constructor and one didn't. I included it in both because I thought that maybe you had an useless constructor there for some obscure reason I didn't know about...